app/Offer/OfferControllerProvider.php

```diff
@@ -63,6 +71,14 @@ public function connect(Application $app): ControllerCollection
 
         $controllers->patch('/{offerType}/{offerId}/', PatchOfferRequestHandler::class);
 
+        /**
+         * Legacy routes that we need to keep for backward compatibility.
+         * These routes usually used an incorrect HTTP method.
```

Comment:
In this case they just have a singular `permission` path.

Reply:
Indeed, that comment was from another file where there used to be more legacy routes and most of those had an incorrect HTTP method that was later changed. But in this case it's a different URL + also slightly different behavior